beancount.el

```diff
@@ -109,6 +109,38 @@ from the open directive for the relevant account."
   `((t :inherit highlight))
   "Face to highlight Beancount transaction at point.")
 
+(defface beancount-level-1
```

Comment:
Can we call this `beancount-heading-1` or `beancount-outline-1` or something else that makes clear that this is about outline and headings?

Reply:
Done.